yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+    datatype: timestamp
+    length: 64
+    description: End of shutter opening.  TAI, accurate to 10ms.
+  - name: shutterCloseBegin
+    "@id": "#Exposure.shutterCloseBegin"
+    datatype: timestamp
+    length: 64
+    description: Start of shutter closing.  TAI, accurate to 10ms.
+  - name: shutterCloseEnd
+    "@id": "#Exposure.shutterCloseEnd"
+    datatype: timestamp
+    length: 64
+    description: End of shutter opening.  TAI, accurate to 10ms.
+  - name: airTemp
+    "@id": "#Exposure.airTemp"
+    datatype: float
+    description: Outside air temperature.
+  - name: pressure
+    "@id": "#Exposure.pressure"
+    datatype: float
+    description: Outside air pressure.
+  - name: humidity
+    "@id": "#Exposure.humidity"
+    datatype: float
+    description: Outside relative humidity.
+  - name: wind_speed
+    "@id": "#Exposure.wind_speed"
+    datatype: float
+    description: Outside wind speed.
+  - name: wind_dir
+    "@id": "#Exposure.wind_dir"
+    datatype: float
+    description: Wind direction.
+  - name: dimm_seeing
+    "@id": "#Exposure.dimm_seeing"
+    datatype: float
+    description: Seeing as measured by external DIMM.
+  - name: shut_lower
+    "@id": "#Exposure.shut_lower"
+    datatype: float
+    description: Dome dropout door opening percentage
+  - name: shut_upper
+    "@id": "#Exposure.shut_upper"
+    datatype: float
+    description: Dome main door opening percentage
+  - name: focus_z
+    "@id": "#Exposure.focus_z"
+    datatype: float
+    description: Focus Z position
+  - name: dome_azimuth
+    "@id": "#Exposure.dome_azimuth"
+    datatype: float
+    description: Dome azimuth
+  - name: simulated
+    "@id": "#Exposure.simulated"
+    datatype: boolean
+    description: Were any control system components simulated?
+  - name: exposureInOODS
+    "@id": "#Exposure.exposureInOODS"
+    datatype: boolean
+    description: Are all detectors ingested into the OODS?
+  - name: exposureInUSDF
+    "@id": "#Exposure.exposureInUSDF"
+    datatype: boolean
+    description: Are all detectors ingested into the USDF?
+- name: CcdExposure
+  "@id": "#CcdExposure"
+  columns:
+  - name: ccdExposureId
+    "@id": "#CcdExposure.ccdExposureId"
+    datatype: long
+    nullable: false
+    description: Primary key (unique identifier).
+  - name: exposure_id
+    "@id": "#CcdExposure.exposure_id"
+    datatype: long
+    nullable: false
+    description: Reference to the corresponding entry in the Exposure table.
+  - name: detector
+    "@id": "#CcdExposure.detector"
+    datatype: long
+    nullable: false
+    description: Detector ID. A detector associated with a particular instrument (not
+      an observation of that detector)
+  - name: xSize
+    "@id": "#CcdExposure.xSize"
+    datatype: long
+    description: Number of columns in the image.
+  - name: ySize
+    "@id": "#CcdExposure.ySize"
+    datatype: long
+    description: Number of rows in the image.
+  - name: llcra
```

Comment:
Do we have code that actually uses the corner positions from this table?  I'm not against having them, but I'm curious about how we use them.